lib/node_modules/@stdlib/math/base/ops/README.md

```diff
@@ -87,10 +87,37 @@ The namespace contains the following functions:
 <!-- eslint no-undef: "error" -->
 
 ```javascript
+var Complex128 = require( '@stdlib/complex/float64' );
 var objectKeys = require( '@stdlib/utils/keys' );
 var ns = require( '@stdlib/math/base/ops' );
 
 console.log( objectKeys( ns ) );
+
+// Operations for floating point numbers (double-precision)
+console.log( ns.add(1.25, 0.45) ); // => 1.7
+console.log( ns.sub(1.25, 0.45) ); // => 0.8
```

Comment:
Console print annotations of the form `// => ...` should always be placed on the following line:

```suggestion
console.log( ns.add(1.25, 0.45) ); 
// => 1.7

console.log( ns.sub(1.25, 0.45) ); 
// => 0.8
```